### PR TITLE
DCJ-445: Update Row Helper Conditional

### PR DESCRIPTION
**Pull request by rushtong**
### Addresses
Partial fix for https://broadworkbench.atlassian.net/browse/DCJ-445

### Summary
This updates minor logic checks for whether a column exists or not. Previous library functionality would have thrown an error, but current functionality does not. JDBI treats null integer values as 0's which are not valid as primary key ids. In those cases, ensure that the value is > 0 before trying to set them on objects.

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by aarohinadkarni — APPROVED** [review]
looks good once tests pass 👍

File: src/main/java/org/broadinstitute/consent/http/db/mapper/DaaMapper.java [hunk 1]
```diff
@@ -15,7 +15,7 @@ public class DaaMapper implements RowMapper<DataAccessAgreement>, RowMapperHelpe
   @Override
   public DataAccessAgreement map(ResultSet resultSet, StatementContext statementContext) throws SQLException {
     DataAccessAgreement daa;
-    Integer daaId = hasColumn(resultSet, "daa_id") ? resultSet.getInt("daa_id") : null;
+    Integer daaId = hasColumn(resultSet, "daa_id") && resultSet.getInt("daa_id") > 0 ? resultSet.getInt("daa_id") : null;
```
Comment (fboulnois) [inline src/main/java/org/broadinstitute/consent/http/db/mapper/DaaMapper.java]: Suggestion: pull this out into a `hasNonZeroColumn` method that these mappers can use.

**Review by fboulnois — COMMENTED** [review]
One suggestion since this conditional is repeated so many times:

File: src/main/java/org/broadinstitute/consent/http/db/mapper/DaaMapper.java (hunk 1 above)
Comment (okotsopoulos) [reply src/main/java/org/broadinstitute/consent/http/db/mapper/DaaMapper.java]: +1 this suggestion.

**Review by okotsopoulos — APPROVED** [review]
Looks good, ditto @fboulnois ' suggestion!

File: src/main/java/org/broadinstitute/consent/http/db/mapper/DaaMapper.java (hunk 1 above)
Comment (rushtong) [reply src/main/java/org/broadinstitute/consent/http/db/mapper/DaaMapper.java]: Great idea, thanks!

**Pull request merged**